otcdocstheme/theme/otcdocs/sidebartoc.html

```diff
@@ -12,3 +12,14 @@
   {%- endif %}
   </div>
 </aside>
+<div class="docs-toc-meta ms-2 pb-1 pt-2 mb-0 d-flex flex-row-reverse d-md-none">
```

Comment:
why do we have it 2 times now? Can we try to join that? otherwise maintenance is going to be complex. Pls also add d-print-none class

Reply:
I have joined both things using include and added the d-print-none class